src/main/java/io/github/mountainrange/mule/GameManager.java

```diff
@@ -261,7 +263,7 @@ public void endTurn() {
 		turnOrder.get(currentPlayerNum).setMule(null);
 		currentPlayerNum = (currentPlayerNum + 1) % config.numOfPlayers;
 
-		resetTimer();
+		turnTimer();
```

Comment:
I don't know why we had resetTimer instead of turnTimer. ResetTimer does not start the timer back up. I will make another pull request to merge these two methods.